core/http/endpoints/openai/video.go

```diff
@@ -0,0 +1,125 @@
+package openai
+
+import (
+    "encoding/json"
+    "fmt"
+    "strconv"
+    "strings"
+
+    "github.com/mudler/LocalAI/core/config"
+    "github.com/mudler/LocalAI/core/http/endpoints/localai"
+    "github.com/mudler/LocalAI/core/http/middleware"
+    "github.com/mudler/LocalAI/core/schema"
+    "github.com/gofiber/fiber/v2"
+    model "github.com/mudler/LocalAI/pkg/model"
+)
+
+func VideoEndpoint(cl *config.ModelConfigLoader, ml *model.ModelLoader, appConfig *config.ApplicationConfig) func(c *fiber.Ctx) error {
```

Comment:
would be great here if we add comments so we get automatically also swagger documentation for the API, consistently with the other API routes.